RELEASE.md

```diff
@@ -0,0 +1,38 @@
+# How to release
+
```

Comment:
When you get ready to do your first release, come chat with me. These docs are a bit outdated and I need to update our RELEASE.md in terra-core to reflect new updates to the process.